### PR TITLE
Update Scale Universe package to 4.4.1

### PR DESCRIPTION
**Pull request by gisjedi**
Update to v4.4.0 of existing Scale package.

@ryadav88 What is the current process to bring a package out of preview and mark as 'selected'? Is this defined?

### CODE REVIEW

**Comment by gisjedi** [comment]
Tested against DCOS 1.9 and discovered that there were issues with the deployment as a result of using an old version of the Marathon Python library. This has been updated and Scale v4.4.1 released to accommodate this change.

Ref: https://github.com/ngageoint/scale/releases/tag/4.4.1

File: repo/packages/S/scale/2/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "remotesensing",
+    "algorithms",
+    "geospatial"
+  ],
+  "framework": true,
+  "version": "4.4.1-1.0.0",
+  "preInstallNotes": "We recommend a minimum of three nodes with at least 4 CPU and 6GB of RAM available for the Scale services and running Scale jobs. By default, Elasticsearch package *must* be running within your DCOS cluster. If you wish to use an externally hosted Elasticsearch cluster, specify one or more of the nodes in comma delimited format in the SCALE_ELASTICSEARCH_URLS variable. For quick-start purposes, Scale is bootstrapped with a Postgres database. This should *NEVER* be used for production purposes as it offers no underlying storage persistence. It can be replaced with an externally hosted Postgres by setting DB_HOST and associated settings appropriately. If you are running DCOS 1.8 Enterprise Edition or higher, you will need to set the DCOS_OAUTH_TOKEN in the DCOS section of the Advanced Settings. This value can be found within the dcos.toml file under in the dcos_acs_token value on a system with an authenticated DCOS CLI.\n\nTutorial documentation for Scale can be found here: https://github.com/dcos/examples/tree/master/scale/1.8",
```
Comment (ryadav88) [inline repo/packages/S/scale/2/package.json]: Add `This DC/OS Service is currently in preview.` at the beginning of `preInstallNotes`

File: repo/packages/S/scale/2/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "postInstallNotes": "The Scale DCOS Service has been successfully installed!\n\nNew User Tutorial: https://github.com/dcos/examples/tree/master/scale/1.8",
```
Comment (ryadav88) [inline repo/packages/S/scale/2/package.json]: change all 1.8 links here to https://github.com/dcos/examples/tree/master/scale

File: repo/packages/S/scale/2/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "postUninstallNotes": "The Scale Scheduler has been uninstalled and will no longer run. You may need to remove supporting services - scale-logstash, scale-webserver and scale-db.",
+  "licenses": [
+    {
+      "url": "https://github.com/ngageoint/scale/blob/master/LICENSE",
+      "name": "Apache License Version 2.0"
+    }
+  ],
+  "description": "Scale enables on-demand, near real-time, automated processing of large datasets (satellite, medical, audio, video, ...) using a dynamic bank of algorithms. Algorithm execution is seamlessly distributed across thousands of CPU cores. Docker provides algorithm containerization. Apache Mesos enables optimum resource utilization.\n\nRelease Notes: https://github.com/ngageoint/scale/releases/tag/4.4.1\n\nDocumentation: https://ngageoint.github.io/scale/\n\nIssues: https://github.com/ngageoint/scale/issues/\n\nNew User Tutorial: https://github.com/dcos/examples/tree/master/scale/1.8",
```
Comment (ryadav88) [inline repo/packages/S/scale/2/package.json]: change example links to https://github.com/dcos/examples/tree/master/scale

**Comment by ryadav88** [comment]
All non-selected packages are marked as `preview`. Product team is working on the exit criteria.

**Comment by gisjedi** [comment]
@ryadav88 Should be good to go. Let me know if anything else is needed.

**Pull request merged**